src/mnemonic-key.ts

```diff
@@ -16,6 +17,28 @@ export class MnemonicKey extends Key {
       mnemonic = Wallet.generateMnemonic()
     }
 
-    this.addByMnemonic(mnemonic, options.index || 0)
+    this._addByMnemonic(mnemonic, options.index || 0, options.defaultPath, options.addresses)
+  }
+
+  private _addByMnemonic(phrase: string, index = 0, defaultPath = '1023', addresses = 1) {
```

Comment:
You need to use a derivationPath, not just a part.

Reply:
Copy!

Reply:
Okay!